### DISCUSSION TITLE
WDIO Chrome driver and standalone server service

### DISCUSSION THREAD

**Discussion by bajipogadadanda** [webdriverio/codemod#104] [General]
Hi,
I am not able to find any documentation or info for WDIO Chrome driver and standalone services.
I have been using these 2 services , and its working fine even today. Just for reference I am in process of get info from wdio docs(https://webdriver.io/docs), but I am not able to find.
I would like to know if any there any change for it, or links are broken.

**Comment by christian-bromann** [comment]
> I have been using these 2 services , and its working fine even today.

They are no needed anymore as WebdriverIO has all capacities to set up the driver for your test. However you can still find docs on the services in e.g. the v7 docs page: https://v7.webdriver.io/docs/selenium-standalone-service
